### DISCUSSION TITLE
First upgrade issue

### DISCUSSION THREAD

**Discussion by mikethebee** [grafana/grafana-csv-datasource#71] [Show and tell]
FYI: Yesterday I updated my Pi OS and got Grafana 7.5.7 and CSV plugin 0.5.0 SVG 0.3.0 after which all dashboard widgets showed "metric request error" After trying a few things, editing each widget by turning 'Header' On and Off again sorted it. 

This may save others some of the time I spent unproductive googling.
-Mike

**Comment by marcusolsson** [comment]
Ouch! Sorry about that 🙏

Do you think you'd be able to list the steps needed to reproduce this in an issue?

**Reply by mikethebee** [reply]
It was really just an OS update, which brought in an update Grafana and then the plugin update (as normal), then all broken widgets. Googling showed that "Metrics Request Error" has been common with Postgres but not elsewise. That is why I note it here. Not a big problem, just weird, Thx

**Reply by mikethebee** [reply]
If it occurs again I will make more notes. It seems I was the only one that experienced it.

**Reply by marcusolsson** [reply]
Only one who reported it at least :) I'll ping you if I hear more on this!

**Reply by mikethebee** [reply]
Just an update, I am still needing to do the header on/off trick after updating the plugin. It is only the Graph visualisation and not on the Gauges. This is on an updated RaspberryPi OS Grafana from the repository.
